modules/nw-udn-cr.adoc

```diff
@@ -0,0 +1,97 @@
+//module included in the following assembly:
+//
+// *networking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-udn-cr_{context}"]
+= Creating a UserDefinedNetwork custom resource
+
+The following procedure sets up a user-defined network that is namespace scoped.
+
+.Prerequisites
+* You must
+.Procedure
+
+[NOTE]
+====
+If any cluster default networked pods exist before the user-defined network is created, any further pods created in this namespace will return an error message: `What_is_this`.
+====
+
+. Create a request for a user-defined network:
+
+.. Create a YAML file, such as `my-udn-request.yaml`, to define your request with content like the following example:
++
+[source, yaml]
+----
+apiVersion: k8s.ovn.org/v1
+kind: UserDefinedNetwork
+metadata:
+  name: udn-1 # <1>
+  namespace: <some_custom_namespace>
+spec:
+  topology: Layer2 # <2>
+  layer2: <3>
+    role: Primary # <4>
+    mtu: 9000 # <5>
```

Comment:
Woth mentioning about MTU boundary as well https://issues.redhat.com/browse/OCPBUGS-38754
Specifically for IPv6 which required min of 1280